src/CoolTensors.jl

```diff
@@ -139,4 +140,13 @@ function Base.typed_hvcat(t::Tensor{<:Any,N,ipos}, alt::Tuple, i...) where {N,ip
     t[I]
 end
 
+using TensorCore
+export tensor, ⊗
+
+function TensorCore.tensor(t::Tensor{<:Any,N,ipos}, s::Tensor{<:Any,M,jpos}) where {N,ipos,M,jpos}
+    ts = tensor(parent(t), parent(s))
+    ijpos = IndexPos{N+M}((ipos.x << M) | jpos.x)
```

Comment:
```suggestion
    ijpos = IndexPos{N+M}((jpos.x << M) | ipos.x)
```

Reply:
Nevermind

Reply:
That's what I did too, but then I think it has to be `N` there as well, hopefully now fixed.